config/gc.json

```diff
@@ -0,0 +1,4 @@
+{
+    "network": "xdai",
+    "startBlock": 16465100
```

Comment:
this PR has changes from other PRs. 
To simplify reviews is best if you select the right base branch

Reply:
I thought I was able to close it, and I rebased it. Sorry for that!

Reply:
no worries 🤗